api/venv/bin/django-admin

```diff
@@ -0,0 +1,11 @@
+#!/Users/phan/ismp/api/venv/bin/python3
```

Comment:
Can you add your venv to gitignore? We can use requirements.txt to keep track of requirements and people can just pip install -r requirements.txt to set up their venvs.

Reply:
I already added to my `.gitignore` -- try rebasing off my latest commit?